### DISCUSSION TITLE
How to ignore live videos

### DISCUSSION THREAD

**Discussion by fqx** [mxpv/podsync#560] [Q&A]
Youtube-dl or yt-dlp will sometimes download part of the live broadcast.
How to stop this behavior and only download when the live is ended?
`youtube_dl_args = ["--match-filter", "!is_live"]`
Is this gonna work?

**Comment by mrbreil** [comment]
this is a good question. i have the same problem

**Comment by fqx** [comment]
@mrbreil you can try my solution, haven’t seen the issue again.

**Comment by dayre** [comment]
These flags worked for me, thanks so much.
